js.es5.md

```diff
@@ -1,6 +1,6 @@
-# K&L JavaScript Style Guide
+# Education ES5 JavaScript Style Guide
```

Comment:
Not sure of the benefit of this? The style guide is just as relevant to ES6. I don't see any ES5-only things in there.